### PR TITLE
Removed custom script to send email. Used mail-send from the mail sender (new) integration.

### PR DESCRIPTION
**Pull request by ankita-sharma2019**
…on in playbook

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Armorblox/Playbooks/Armorblox_Needs_Review.yml [hunk 1]
```diff
@@ -203,31 +175,130 @@ tasks:
...
+    note: false
+    quietmode: 0
+    scriptarguments:
+      value:
+        simple: Please configure Mail Sender (New) Integration to enable this feature
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Prints text to war room (Markdown supported)
```
Comment (urajat) [inline Packs/Armorblox/Playbooks/Armorblox_Needs_Review.yml:276]: I see 2 instances of this text. Is this relevant to us?

**Review by urajat — APPROVED** [review]

File: Packs/Armorblox/Playbooks/Armorblox_Needs_Review.yml (hunk 1 above)
Comment (ankita-sharma2019) [reply Packs/Armorblox/Playbooks/Armorblox_Needs_Review.yml:276]: Are you talking about line 276?
